README.rst

```diff
@@ -653,11 +656,80 @@ using the ``pypath.settings`` module.
 The ``pypath.session`` and ``pypath.log`` modules take care of setting up
 session level parameters and logging. Each session has a random 5 character
 identifier e.g. ``y5jzx``. The default log file in this case is
-``pypath_log/pypath-y5jzx.log``. The log messages flushed in every 2 seconds
+``pypath_log/pypath-y5jzx.log``. The log messages are flushed in every 2 seconds
 by default. You can always change these things by the ``settings`` module.
-In this module you can get and set the values of various parameters using
+In this module, you can get and set the values of various parameters using
 the ``pypath.settings.setup()`` and the ``pypath.settings.get()`` methods.
 
 A simple **webservice** comes with this module: the ``server`` module based on
 ``twisted.web.server`` opens a custom port and serves plain text tables over
 HTTP with REST style querying.
+
+Developers' corner
```

Comment:
It's up to you. However, I think we should leave this section somewhere else which might be handy for the end-users wanting to use the package instead of developing packages.

Reply:
Then again, end-users are free to use the package in any IDE they feel like. And if intended for "end-users wanting to use the package instead of developing" then why call it "Developers' corner"?